cg/store/models.py

```diff
@@ -987,3 +992,59 @@ class Order(Base):
 
     def to_dict(self):
         return to_dict(model_instance=self)
+
+
+class DigitisationUnit(Base):
+    """Parent model representing the consumable units (cells/chips) on which the sample data is generated."""
+
+    __tablename__ = "digitisation_unit"
+
+    id: Mapped[PrimaryKeyInt]
+    type: Mapped[MachinesTypes]
+    model: Mapped[Str32]
+    internal_id: Mapped[UniqueStr]
+
```

Comment:
```suggestion
batch_id: Mapped[int] = mapped_column(ForeignKey("RunBatch.id"))
```
We want to add this here s well?